packages/callout/src/Callout.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { renderToStaticMarkup } from 'react-dom/server';
+import { cleanup, render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import Callout, {
+  colorSets,
+  headerIcons,
+  headerLabels,
+  CustomCallout,
+  CustomCalloutProps,
+  Variant,
+} from './Callout';
+
+const className = 'test-classname';
+const children = 'this is the callout content.';
+const title = 'this is the callout title';
+const colorSet = colorSets[Variant.Note];
+const headerIcon = headerIcons[Variant.Note];
+const headerLabel = headerLabels[Variant.Note];
+
+function renderCustomCallout(props: Partial<CustomCalloutProps> = {}) {
+  const defaultProps = {
+    children,
+    colorSet,
+    headerLabel,
+  };
+  return render(<CustomCallout {...defaultProps} {...props} />);
+}
+
+describe('packages/callout', () => {
+  describe('Callout component', () => {
+    test('renders each variant correctly', () => {
+      for (const variant of Object.values(Variant)) {
+        const calloutMarkup = renderToStaticMarkup(
```

Comment:
Have never used the `renderToStaticMarkup` helper before, will need to check it out but this test suite looks great :)